src/Component/Histories/Restaurant/Restaurant.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { getExternalRestaurantUrl } from '../../../Util'
+
+// Restaurant :: Props -> React.Component
+export default ({
+  restaurant,
+  history,
```

Comment:
if you only need to use the restaurantId property better to just pass that property instead of all the history object.
In case only one property (different from restaurantId) changes on the history object React will re-render this component even if the result will be the same.

Reply:
I fixed it differently, by removing the `restaurant` variable one and use the history to get it :')